### DISCUSSION TITLE
Login error with Google SSO

### DISCUSSION THREAD

**Discussion by fresquer** [outline/outline#4022] [Self Hosting]
Hi,

I've enabled google as my SSO and I can't use any email domain execept the domain I've use to create my admin account. 

I tried my google auth api in other project and works like a charm. All the domains are configured in the same google workspace adminstration. The one which works is the principal and other ones are "secondary domains". Could be this the problem?

The console error:
`{"error":"_invariant is not defined","stack":"UnauthorizedError: _invariant is not defined\n    at AuthenticationError (/root/outline/build/server/errors.js:35:34)\n    at accountProvisioner (/root/outline/build/server/commands/accountProvisioner.js:45:43)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Strategy._verify (/root/outline/build/server/routes/auth/providers/google.js:67:18)","level":"error","message":"Error during authentication"}`

Thanks for your help!

**Comment by tommoor** [comment]
Which version of Outline are you using

**Reply by fresquer** [reply]
The version is 0.64.3

**Reply by tommoor** [reply]
First step is to upgrade to the latest version then and see if the error continues
